src/main/java/com/vaadin/training/bugrap/ui/component/CommentPanel.java

```diff
@@ -2,27 +2,33 @@
 
 import java.util.Date;
 
+import org.vaadin.bugrap.domain.entities.Comment;
+
 import com.vaadin.training.bugrap.util.ElapsedTimeFormat;
 import com.vaadin.ui.CustomComponent;
 
-public class Comment extends CustomComponent {
+public class CommentPanel extends CustomComponent {
 
 	private static final long serialVersionUID = -7113423443746306442L;
 
 	private static final String AUTHOR_TEMPLATE = "%s (%s)";
 	private static final String ANONYMOUS = "Anonymous";
 
-	private final CommentDesign commentDesign;
+	private final CommentPanelDesign commentDesign;
```

Comment:
This seems a bit redundant. Perhaps better way would be to make CommentPanel extend CommentPanelDesign and extend from CustomComponent when there is no design-class available

Reply:
Ok!